hexa/databases/api.py

```diff
@@ -100,6 +101,16 @@ def create_database(db_name: str, pwd: str):
             cursor.execute("create extension postgis;")
             cursor.execute("create extension postgis_topology;")
 
+            with open(
+                os.path.join(os.path.dirname(__file__), "static/demo.sql")
+            ) as file:
+                cursor.execute(file.read())
+                cursor.execute(
+                    sql.SQL(
+                        "GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public TO {role_name};"
```

Comment:
I think it would be easier to change the ownership of the table. Could you try with `ALTER TABLE table_name OWNER TO {role_name};`?

Reply:
Yes, you right.

Reply:
I'd move the upload of the demo data in a different function that would be called after this one as this function is not aware of the workspace.
WDYT @pvanliefland ?

Reply:
Agreed with @qgerome .

@cheikhgwane I know that you technically don't have access to the table name but you know which table the SQL file are creating. I'd prefer that the role owns all the tables in the workspace.